torch_frame/data/mapper.py

```diff
@@ -395,3 +396,31 @@ def forward(
 
     def backward(self, tensor: Tensor) -> pd.Series:
         raise NotImplementedError
+
+
+class EmbeddingTensorMapper(TensorMapper):
+    r"""Maps embedding columns into tensors."""
+    def forward(
+        self,
+        ser: Series,
+        *,
+        device: Optional[torch.device] = None,
+    ) -> MultiEmbeddingTensor:
+        # NOTE We are converting the default PyTorch dtype into a string
+        # representation that can be understood by numpy.
+        # TODO Think of a "less hacky" way to do this.
+        dtype = str(torch.get_default_dtype()).split('.')[-1]
```

Comment:
For embedding, the dtype should always be float32?

Reply:
Maybe we should keep this if user want to train the model with float16?